doc/release-notes/0.15/0.15.md

```diff
@@ -120,13 +117,27 @@ prints statistics about classes on the heap, including number of objects and agg
 </tr>
 
 <tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5374">#5374</a></td>
-<td valign="top">New heuristics to implement compaction during idle garbage collection.</td>
+<td valign="top">New heuristics to implement compaction during idle garbage collection</td>
 <td valign="top">OpenJDK8 and later</td>
 <td valign="top">Heuristics are added to automatically compact the heap during idle garbage collection (GC) processing. Therefore, if
 <tt>-XX:+IdleTuningGCOnIdle</tt> is enabled, compaction might occur. To force compaction every time idle GC operations take place,
 <tt>-XX:+IdleTuningCompactOnIdle</tt> must be enabled. </td>
 </tr>
 
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5374">#5374</a></td>
+<td valign="top">Change in shared classes behavior for checking timestamps of jar or zip files</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In earlier releases, the shared classes cache checks timestamps of jar or zip files every time a class is loaded and reloads a class if the timestamp has changed. This behavior is now changed; timestamps are checked only when zip or jar files are added to class loaders and used for the first time to look for a class.
+To revert to the behavior of earlier releases, set the <tt>-Xshareclasses:checkURLTimestamps</tt> option on the command line when you start your application. </td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/6059">#6059</a></td>
+<td valign="top">Automatically setting an initial heap size</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">OpenJ9 can now learn and set an appropriate initial heap size for an application as an alternative to a user manually sizing and setting an <tt>-Xms</tt> value. The VM records the size of the heap when startup processing ends, writing this data to the shared classes cache. An average value is set over ~5 restarts, helping to ensure that the value used for the initial heap size is as accurate as possible. See the <tt>-XX:[+|-]useGCStartupHints</tt> option (disabled by default). The hint
```

Comment:
I merged this already, but Alek's comment about using `5` should be address here as well.